file-formats/clas/type/zif_aff_clas_v1.intf.abap

```diff
@@ -74,9 +74,18 @@ INTERFACE zif_aff_clas_v1 PUBLIC.
       "! <p class="shorttext">Message Class</p>
       "! The message class of the class
       message_class        TYPE c LENGTH 20,
-      "! <p class="sharttext"> Descriptions</p>
-      "! Descriptions maintained in SE80
-      descriptions         TYPE  zif_oo_aff_types_v1=>ty_descriptions,
+      "! <p class="shorttext">Type Descriptions</p>
+      "! Type descriptions
+      types                TYPE zif_oo_aff_types_v1=>ty_component_descriptions,
+      "! <p class="shorttext">Attribute Descriptions</p>
+      "! Attribute descriptions
+      attributes           TYPE zif_oo_aff_types_v1=>ty_component_descriptions,
+      "! <p class="shorttext">Event Descriptions</p>
+      "! Event descriptions
+      events               TYPE zif_oo_aff_types_v1=>ty_events,
+      "! <p class="shorttext">Method Descriptions</p>
+      "! Method descriptions
+      methods              TYPE zif_oo_aff_types_v1=>ty_methods,
```

Comment:
looks like this reverts the changes from https://github.com/SAP/abap-file-formats/pull/124

Reply:
thanks, that has to be changed